growth/too/views.py

```diff
@@ -989,8 +1002,8 @@ def get_json_data_manual(form):
                       'dec': field.dec,
                       'filter_id': filter_id,
                       'exposure_time': exposure_time,
-                      'program_pi': program_pis[telescope],
-                      'subprogram_name': "ToO_manual"
+                      'program_pi': program_pis[telescope] + '/' + username,
+                      'subprogram_name': "ToO_" + subprogram_name
```

Comment:
Could you please add the subprogram name choices underneath (like for the filters), i.e. GW, GRB, Neutrino?